Car-slider.html

```diff
@@ -0,0 +1,238 @@
+        margin: 0;
+        padding: 0;
+        outline: none;
+      }
+
+      .car__info__wrapper {
+        display: inline-block;
+        padding-left: 5px;
+        padding-right: 5px;
+      }
+
+      .card__image__content {
+        background-color: rgb(248, 246, 246);
+        border-radius: 5px;
+        border: 1px solid rgb(202, 202, 202);
+      }
+
+      .card__container {
+        width: 63%;
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        border-radius: 5px;
+      }
+
+      .car__information {
+        padding: 16px 12px 20px;
+      }
+
+      .car__name {
+        font-size: 16px;
+        color: #484848;
+        font-weight: 400;
+      }
+
+      .car__price {
+        font-size: 16px;
+        margin-top: 8px;
+        font-weight: 700;
+        color: #484848;
+      }
+
+      .car__avg__price {
+        font-size: 13px;
+        margin-top: 8px;
+        color: #6f6f6f;
+      }
+
+      .car__city__price {
+        margin-top: 8px;
+      }
+
+      .citywise__price {
+        color: #0288d1;
+        text-decoration: none;
+      }
+
+      .onwards__text {
+        font-size: 13px;
+        font-weight: 400;
+        color: #aaa;
+        padding-top: 2px;
+        padding-left: 3px;
+      }
+
+      .previous__card__slider,
+      .next__card__slider {
+        height: 32px;
+        width: 32px;
+        background-color: #d7d7d7;
+        position: absolute;
+        z-index: 1;
+        top: 50%;
+        border-radius: 50%;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+
+      .next__card__slider {
+        right: 0;
+      }
+
+      .previous__arrow {
+        transform: rotate(180deg);
+      }
+
+      .previous__arrow,
+      .next__arrow {
+        width: 16px;
+        height: 16px;
+        color: #484848;
+      }
+
+      .car__slider__wrapper {
+        display: flex;
+        overflow-x: scroll;
+      }
+
+      ::-webkit-scrollbar {
+        width: 4px;
+        height: 1px;
+      }
+      ::-webkit-scrollbar-thumb {
+        background-color: #c5c5c5;
+        border-radius: 12px;
+        border: 1px solid #f6f6f6;
+      }
+      ::-webkit-scrollbar-track {
+        background: #f6f6f6;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="card__container">
+      <a class="previous__card__slider" href="#box1">
+        <svg
+          class="previous__arrow"
+          viewBox="0 0 16 16"
+          fill="currentcolor"
+          tabindex="-1"
+          focusable="false"
+          aria-hidden="true"
+          role="img"
+        >
+          <path
+            d="M4.59 15.8a1 1 0 01-.69-.29 1 1 0 010-1.37L10 8 3.9 1.86a1 1 0 010-1.37 1 1 0 011.38 0l6.82 6.82a1 1 0 010 1.38l-6.82 6.82a1 1 0 01-.69.29z"
+          ></path>
+        </svg>
+      </a>
+
+      <div class="car__slider__wrapper">
+        <div class="car__info__wrapper" id="box1">
```

Comment:
```suggestion
        <div class="card" id="box1">
```
can be assumed as another block